doc/source/intro.rst

```diff
@@ -47,7 +47,7 @@ relationships of |resources|..
 .. _MySQL: http://mysql.org
 
 Architecture overview
----------------------
+~~~~~~~~~~~~~~~~~~~~~
```

Comment:
tbh, it feels strange this is the same level as drivers... i feel like drivers should be lower